src/test/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/SurveySvcRestClientImplTest.java

```diff
@@ -37,28 +44,24 @@
 
   @Mock private RestTemplate restTemplate;
 
-  @Spy private RestUtility restUtility = new RestUtility(RestUtilityConfig.builder().build());
+  @Spy private final RestUtility restUtility = new RestUtility(RestUtilityConfig.builder().build());
```

Comment:
I know you've just made this final, but I don't understand why this wouldn't be mocked... we don't want our unit tests attempting to do REST requests